website/ref/language.md

```diff
@@ -2019,6 +2019,15 @@ recursive functions:
 Under the hood, `fn` defines a variable with the given name plus `~` (see
 [variable suffix](#variable-suffix)).
 
+```elvish-transcript
+~> fn f []{ echo hello from f }
+~> f
+hello from f
+~> ff = $f~
```

Comment:
I'd name this variable to be completely different so that it's a bit easier to read. Right now there are too many `f`s.